### ISSUE TITLE
Add support for fetching code with fetchcode in a pipeline

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode.io#35]
This would help downloading files on demand in a pipeline. This would require that https://github.com/nexB/fetchcode/ be packaged for reuse. @TG1999 FYI

**Comment by TG1999** [comment]
Agreed @pombredanne , Readme of fetchcode is done, setup files are required, working on it, so we can publish fetchcode on pypi :)

**Comment by tdruez** [comment]
@TG1999 What's the status on the fetchcode availability?

**Comment by TG1999** [comment]
I have created a PR for this here https://github.com/nexB/fetchcode/pull/46, CI is failing with new changes, earlier all tests were passing, I need some help over that
